src/Confluent.Kafka/LogLevel.cs

```diff
@@ -0,0 +1,41 @@
+namespace Confluent.Kafka
+{
+    /// <summary>
+    ///  Represents enumeration with levels comming from syslog(3)
+    /// </summary>
+    public enum LogLevel
+    {
+        /// <summary>
+        ///  debug-level message
+        /// </summary>
+        Debug,
```

Comment:
these need to be explicitly mapped to syslog error levels, e.g. Debug = 7, ..

Reply:
Hi @edenhill , thanks for the review. I will map them accordingly I assume that this is the desired mapping?
```
0 | EMERGENCY
1 | ALERT 
2 | CRITICAL 
3 | ERROR 
4 | WARNING
5 | NOTICE 
6 | INFORMATIONAL
7 | DEBUG
```

Reply:
Yep, looks good